tools/sdk/include/user_interface.h

```diff
@@ -531,7 +541,7 @@ typedef struct {
 
 typedef struct {
     uint8 mac[6];
-    struct ip_addr ip;
+    struct ipv4_addr ip;
```

Comment:
What happens with this event with ipv6?
How does ipv6 work with SoftAP? What happens with the clients that connect to the SoftAP?

Reply:
SoftAP is IPv4 for the FW structures. Everything is not tested. IPv6 could be used but we should first upgrade the DHCP server which is still espressif-patched-lwIP-v1.4's (even with lwIP-v2 enabled).

Reply:
Does that mean that this even will never trigger with IPv6, because of lack of dhcp6?